packages/commons/src/risk-analysis/riskAnalysisValidation.ts

```diff
@@ -30,21 +30,30 @@ import { riskAnalysisFormRules } from "./rules/riskAnalysisFormRulesProvider.js"
 export function validateRiskAnalysis(
   riskAnalysisForm: RiskAnalysisFormToValidate,
   schemaOnlyValidation: boolean,
-  tenantKind: TenantKind
+  tenantKind: TenantKind,
+  dateForValidation: Date
```

Comment:
Are there cases where we want to check expiration against a date that is not the current instant?
If not, I suggest we remove this param and just use `new Date()` here in the check. Alternatively, we can default this to `new Date()` so we don't have to pass it everywhere.
Also, I suggest a more specific name like `dateForExpirationValidation`

```suggestion
  dateForExpirationValidation: Date = new Date()
```